src/main/kotlin/org/rust/ide/colors/RustColors.kt

```diff
@@ -9,11 +9,11 @@ import com.intellij.openapi.editor.DefaultLanguageHighlighterColors as Default
  */
 enum class RustColor(humanName: String, val default: TextAttributesKey) {
     IDENTIFIER            ("Identifier",                  Default.IDENTIFIER),
-    FUNCTION_DECLARATION  ("Function declaration",        Default.FUNCTION_DECLARATION),
-    INSTANCE_METHOD       ("Instance method declaration", Default.INSTANCE_METHOD),
-    STATIC_METHOD         ("Static method declaration",   Default.STATIC_METHOD),
-    PARAMETER             ("Function parameter",          Default.PARAMETER),
-    SELF_PARAMETER        ("Self parameter",              Default.KEYWORD),
+    FUNCTION_DECLARATION  ("Function",                    Default.FUNCTION_DECLARATION),
+    INSTANCE_METHOD       ("Method",                      Default.INSTANCE_METHOD),
+    STATIC_METHOD         ("Associated function",         Default.STATIC_METHOD),
+    PARAMETER             ("Function argument",           Default.PARAMETER),
```

Comment:
Let's change "argument" to 'parameter". This is not really cast in stone, but usually when people refer to "parameter", they mean the formal parameters of function *declaration* `fn foo(i_am_a_format_parameter: i32)`. And "argument" is usually the actual value that is passed to the function *call* `foo(1 + 2 + 3)`.